draft-privacypass-rate-limit-tokens.md

```diff
@@ -859,9 +839,9 @@ policy if supported. If a cross-origin policy is not supported, this condition i
 If any of these conditions is not met, the Issuer MUST return an HTTP 400 error to the Attester,
 which will forward the error to the client.
 
-The Issuer determines the correct Issuer Key by using the decrypted Origin Name value and
-TokenRequest.token_key_id. If there is no Token Key whose truncated key ID matches
-TokenRequest.token_key_id, the Issuer MUST return an HTTP 401 error to Attester, which will
+The Issuer determines the correct Issuer Key by using the decrypted Origin Name and
+InnerTokenRequest.token_key_id values. If there is no Token Key whose truncated key ID matches
```

Comment:
Probably want to explicitly call out that the token key has to match the key ID and origin name, to prevent cross-origin key usage.

"If there is no Token Key whose truncated key ID matches InnerTokenRequest.token_key_id and whose origin matches Origin Name, ..."